hs_core/views/__init__.py

```diff
@@ -454,10 +514,34 @@ def my_resources(request, page):
             'first': start,
             'last': start+len(res),
             'ct': total_res_cnt,
+            'dcterms' : (
+                ('AB', 'Abstract'),
+                ('BX', 'Box'),
+                ('CN', 'Contributor'),
+                ('CVR', 'Coverage'),
+                ('CR', 'Creator'),
+                ('DT', 'Date'),
+                ('DTS', 'DateSubmitted'),
+                ('DC', 'DateCreated'),
+                ('DM', 'DateModified'),
+                ('DSC', 'Description'),
+                ('FMT', 'Format'),
+                ('ID', 'Identifier'),
+                ('LG', 'Language'),
+                ('PD', 'Period'),
+                ('PT', 'Point'),
+                ('PBL', 'Publisher'),
+                ('REL', 'Relation'),
+                ('RT', 'Rights'),
+                ('SRC', 'Source'),
+                ('SUB', 'Subject'),
+                ('T', 'Title'),
+                ('TYP', 'Type'),
+    )
```

Comment:
I addressed this by checking out init:

```
git checkout 73643b70e3355626ba4993e07649815dd2746c5c -- hs_core/views/__init__.py
```
Why isn't this updated when I pull develop?